src/main/kotlin/StringCalculator.kt

```diff
@@ -0,0 +1,25 @@
+import ExceptionCase.Companion.CASE_INCORRECT_INPUT
+import ExceptionCase.Companion.CASE_INPUT_NULL_OR_EMPTY
+import ExceptionCase.Companion.CASE_INT_FORMAT_OR_NULL
+
+/**
+ * 사측연산 계산 class
+ * */
+class StringCalculator {
+    fun calculate(input: String?): Int? {
+        if (input.isNullOrBlank()) throw IllegalArgumentException(CASE_INPUT_NULL_OR_EMPTY)
```

Comment:
```suggestion
       require(!input.isNullOrBlank()){CASE_INPUT_NULL_OR_EMPTY}
```
https://jinee0717.tistory.com/10 참고해보세요:)